roborazzi-compose/src/main/java/com/github/takahirom/roborazzi/RoborazziComposePreview.kt

```diff
@@ -117,7 +124,24 @@ private fun (@Composable () -> Unit).withSize(
       heightDp > 0 -> Modifier.height(heightDp.dp)
       else -> Modifier
     }
-    Box(modifier = modifier) { this@withSize() }
+    Box(modifier = modifier) {
+      this@withSize()
+    }
   }
   return resizedPreview
+}
+
+private fun registerRoborazziComposePreviewsActivityToRobolectricIfNeeded() {
+  try {
+    val appContext: Application = ApplicationProvider.getApplicationContext()
+    Shadows.shadowOf(appContext.packageManager).addActivityIfNotPresent(
+      ComponentName(
+        appContext.packageName,
+        ComponentActivity::class.java.name,
+      )
+    )
+  } catch (e: ClassNotFoundException) {
```

Comment:
I'm curious about this.

You are directly calling both robolectric and ComponentActivity. Wouldn't it fail with an error (NoClassDefFoundError , or similar) rather than a ClassNotFoundException?  I normally assume ClassNotFoundException is for reflection usage with a checked exception.

Reply:
I agree and haven’t realised that since I copy pasted it from the analogue method in RoborazziCompose.kt file.

worth checking 👍